src/hooks/useErrorModal.ts

```diff
@@ -0,0 +1,23 @@
+import { useModal } from "components/UI/Modals/GenericModal/GenericModalProvider";
+import { useEffect } from "react";
+
+export const useErrorModal = (
+  errorCondition: boolean,
+  beforeClose: (...args: any) => any,
+  errorMessage?: string
+) => {
+  const { openModal, closeModal } = useModal();
+
+  useEffect(() => {
+    if (errorCondition) {
+      openModal("errorModal", {
+        visible: true,
```

Comment:
remove, when press `openModal` don't need to send `visible: true`.

Reply:
forgot to remove it, fixed now